tests/tests/fv3_rrfs_v1beta_debug

```diff
@@ -14,30 +14,30 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
                    atmos_4xdaily.tile4.nc \
                    atmos_4xdaily.tile5.nc \
                    atmos_4xdaily.tile6.nc \
-                   phyf000.tile1.nc \
-                   phyf000.tile2.nc \
-                   phyf000.tile3.nc \
-                   phyf000.tile4.nc \
-                   phyf000.tile5.nc \
-                   phyf000.tile6.nc \
-                   phyf003.tile1.nc \
-                   phyf003.tile2.nc \
-                   phyf003.tile3.nc \
-                   phyf003.tile4.nc \
-                   phyf003.tile5.nc \
-                   phyf003.tile6.nc \
-                   dynf000.tile1.nc \
-                   dynf000.tile2.nc \
-                   dynf000.tile3.nc \
-                   dynf000.tile4.nc \
-                   dynf000.tile5.nc \
-                   dynf000.tile6.nc \
-                   dynf003.tile1.nc \
-                   dynf003.tile2.nc \
-                   dynf003.tile3.nc \
-                   dynf003.tile4.nc \
-                   dynf003.tile5.nc \
-                   dynf003.tile6.nc \
+                   sfcf000.tile1.nc \
```

Comment:
If changes are made to the debug version of the rrfs v1beta test, why not to the prod version? How about the recently added rrfs v1 alpha? Have you checked that all tests have been updated with this prefix change?

Reply:
All the rrfs tests are now using "sfcf/atmf" in the history file names. I updated most global tests except those with hard-coded "dyn/phy" in their own version of model_configure files

Reply:
Thanks, that's great.